mail_tracking/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Email tracking",
     "summary": "Email tracking system for all mails sent",
-    "version": "8.0.3.0.0",
+    "version": "8.0.3.0.1",
```

Comment:
No update is needed, and if you do not change anything in ir.config_parameter, then your addon will run exactly in the same way. That's why I use the last number.